sherpa/astro/ui/utils.py

```diff
@@ -12401,13 +12409,19 @@ def save_all(self, outfile=None, clobber=False):
         Notes
         -----
 
-        Items which are not saved include:
+        This command will create a series of commands that restores
+        the current Sherpa set up. It does not save the set of commands
+        used. Not all Sherpa settings are saved. Items not fully restored
+        include:
 
-        - user models
+        - data created by calls to `load_arrays`, or changed from the
+          version on disk - e.g. by calls to `sherpa.astro.ui.set_counts`.
 
         - any optional keywords to comands such as `load_data`
           or `load_pha`
 
+        - user models may not be restored correctly
+
         - only a subset of Sherpa commands are saved.
```

Comment:
This is rather vague, for a section that should indeed describe what is missing. This is not an issue of the PR per se, as this part is unchanged from the current code, but I thought it was worth mentioning, as maybe it could be fixed at this round.